app/org/sagebionetworks/bridge/upload/UploadValidationContext.java

```diff
@@ -26,6 +27,16 @@
     private Map<String, byte[]> attachmentsByFieldName;
     private String recordId;
 
+    /** Health code of the user contributing the health data. */
+    public String getHealthCode() {
+        return healthCode;
```

Comment:
I don't think so, for several reasons:

1. Extra logic we'd have to maintain and test. It would be better for a simple getter to remain a simple getter. getUploadId() exists primarily because it's a common enough construct that it's worth having rather than adding a bunch of null checks everywhere.
2. Health Code exists independently of Uploads. We were originally getting it from the Upload probably because we were too lazy to make it its own top-level field. In contrast, Upload ID is meaningful only when there's an Upload.